doc/release-process.md

```diff
@@ -40,23 +40,33 @@ Check out the source code in the following directory hierarchy.
     git clone https://github.com/devrandom/gitian-builder.git
     git clone https://github.com/bitcoin/bitcoin.git
 
-### Bitcoin maintainers/release engineers, suggestion for writing release notes
+### Write the release notes
 
-Write release notes. git shortlog helps a lot, for example:
+Open a draft of the release notes for collaborative editing at https://github.com/bitcoin-core/bitcoin-devwiki/wiki.
+
+For the period during which the notes are being edited on the wiki, the version on the branch should be wiped and replaced with a link to the wiki which should be used for all announcements until `-final`.
```

Comment:
The steps to be done pre-branch-off should be mentioned above, no?

See the commit in #15692, which you are welcome to steal.

Reply:
Ok :+1:

Reply:
Stolen, adapted/merged, rebased.